$MirOS: ports/databases/postgresql/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:44:16 tg Exp $
--- configure.in.orig	Wed Feb  7 04:48:35 2007
+++ configure.in	Thu Feb 22 20:32:36 2007
@@ -59,6 +59,7 @@ case $host_os in
  linux*|gnu*|k*bsd*-gnu)
            template=linux ;;
    mingw*) template=win32 ;;
+  mirbsd*) template=openbsd ;;
   netbsd*) template=netbsd ;;
 nextstep*) template=nextstep ;;
  openbsd*) template=openbsd ;;
@@ -577,7 +578,7 @@ AC_ARG_VAR(LDFLAGS_SL)
 AC_PROG_AWK
 PGAC_PATH_FLEX
 AC_PROG_LN_S
-PGAC_PROG_LD
+AC_PROG_LD
 AC_SUBST(LD)
 AC_SUBST(with_gnu_ld)
 case $host_os in sysv5*)
